_posts/en/2021-05-18-problems-of-project-based-funding.md

```diff
@@ -0,0 +1,131 @@
+
+## Some Data
+
+In [ESDC's Project Centre](https://gpp-ppm.service.gc.ca/sites/pwa/Projects.aspx) we see that there are 103 IT-Enabled projects in progress[^1], with 101 of them requiring a Treasury Board Submission (TB sub).
+Not all projects are large in nature, in fact 61% of the 101 TB subs are for projects under $5M, with 29% being for under $500k.
+
+Small projects are good.
+As per the [Standish Group’s CHAOS reports](https://standishgroup.com/sample_research_files/CHAOSReport2015-Final.pdf), small IT-Enabled projects have a much higher resolution success rate (resolution = on time, on budget, with satisfactory results).
+But we believe the burden of our current funding mechanism [inadvertently promotes risky behaviours](https://sara-sabr.github.io/ITStrategy/2019/12/20/why-we-are-promoting-risks.html).
+
+A sample of 26 IT-Enabled projects[^2] shows that it takes, on average, **1,291 days for them to start their execution stage**.
+Meaning the IT organization, on average, starts development and testing of the IT solution only after 3.5 years from the project’s inception.
+
+## The effects projects have on the IT organization
+
+We understand that TB subs are necessary for obtaining expenditure approval and additional funding[^3].
+In addition, because software is essentially a collaborative process involving many teams[^4], projectizing the work is often seen as  necessary to formalize engagements between individual IT teams and coordinating that work to completion.
+
+Projects are therefore used to compete for funds.
+Funds used for salary and non-salary expenditures of multiple IT teams.
+In a sense, projects are used to compete for the IT staff’s prioritization of work.
+The problem happens when you have a large amount of advance planning, bundling of various work affecting large scale software applications, and the inevitable quarterly re-prioritization, all colliding on IT teams.
+An IT team responsible for a product is in a constant struggle to re-adjust its work based on changing demands in addition to providing long-term costing estimates to these individual projects.
+This product team will spend a great deal of energy on planning expectations of work with other peer IT teams[^5] to be able to fulfil commitments it made to all these projects.
+
+<div style="display: block; width: inherit; text-align: center; line-height: 3;">
+    <a href="/ITStrategy/assets/images/project-planning-effect-on-product-teams-en.png" style="display: block">
+        <img src = "{{ site.baseurl }}/assets/images/project-planning-effect-on-product-teams-en.png"
+        alt = "This image depicts the cognitive load of multiple projects hitting a product team, and that product team needing to coordinate and plan with other IT teams (that are part of the value stream). Ultimately, the product team is being asked to provide costing and scheduling estimates for each individual projects. Providing this information is also work."
```

Comment:
I would suggest slightly different alt text.  The text in the 'Figure 1' statement describes what the image is depicting, the alt text should describe what the image looks like.  

"This image has a silhouette of a project manager in the top left. To the right of the project manager silhouette there are 8 icons displayed in a vertical line representing 8 different projects.  There are three arrows going from the vertical line of project icons towards a sideways profile of a head. The top of the head is opened as if on a hinge and to the right of the head there are 4 icons arranged in a vertical line of a person carrying a heavy box. There are coloured circles going from them to inside the open head.  Underneath the open head are the words 'Product team'. Underneath the 4 icons to the right of the head are the words 'Value stream teams'."